### PR TITLE
[webgpu] Add ceil, floor, expm1, rsqrt

### PR DESCRIPTION
**Pull request by axinging**
FEATURE

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4752)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @xhcao PTAL

**Comment by qjia7** [comment]
LGTM
@lina128 take a look, thanks.

File: tfjs-backend-webgpu/src/kernels/Ceil.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const CEIL = `return ceil(a);`;
+
+export const ceil =
+    unaryKernelFunc({opSnippet: CEIL, cpuKernelImpl: ceilImplCPU});
+
+export const ceilConfig: KernelConfig = {
+  kernelName: Ceil,
+  backendName: 'webgpu',
+  kernelFunc: ceil as {} as KernelFunc
```
Comment (gyagp) [inline tfjs-backend-webgpu/src/kernels/Ceil.ts:30]: This is unnecessary.

File: tfjs-backend-webgpu/src/kernels/Expm1.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+const EXPM1 = `return exp(a) - 1.0;`;
+
+export const expm1 =
+    unaryKernelFunc({opSnippet: EXPM1, cpuKernelImpl: expm1ImplCPU});
+
+export const expm1Config: KernelConfig = {
+  kernelName: Expm1,
+  backendName: 'webgpu',
+  kernelFunc: expm1 as {} as KernelFunc
```
Comment (gyagp) [inline tfjs-backend-webgpu/src/kernels/Expm1.ts:30]: This is unnecessary.

File: tfjs-backend-webgpu/src/kernels/Rsqrt.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+const RSQRT = `return inversesqrt(a);`;
+
+export const rsqrt =
+    unaryKernelFunc({opSnippet: RSQRT, cpuKernelImpl: rsqrtImplCPU});
+
+export const rsqrtConfig: KernelConfig = {
+  kernelName: Rsqrt,
+  backendName: 'webgpu',
+  kernelFunc: rsqrt as {} as KernelFunc
```
Comment (gyagp) [inline tfjs-backend-webgpu/src/kernels/Rsqrt.ts:30]: This is unnecessary.

File: tfjs-backend-webgpu/src/register_all_kernels.ts [hunk 4]
```diff
@@ -47,9 +50,9 @@ import {lessEqualConfig} from './kernels/LessEqual';
 import {logConfig} from './kernels/Log';
 import {maxConfig} from './kernels/Max';
 import {maximumConfig} from './kernels/Maximum';
-import {minimumConfig} from './kernels/Minimum';
 import {maxPoolConfig} from './kernels/MaxPool';
 import {minConfig} from './kernels/Min';
+import {minimumConfig} from './kernels/Minimum';
```
Comment (gyagp) [inline tfjs-backend-webgpu/src/register_all_kernels.ts:55]: Please also change the sequence in kernelConfigs.

**Comment by gyagp** [comment]
LGTM, @lina128 please take a look!

**Review by lina128 — APPROVED** [review]
LGTM!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4752#-:-MV0Jqx40Sb_-fsR0oT6:b-85i648)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @axinging)

**Pull request merged**